src/SURL.h

```diff
@@ -0,0 +1,44 @@
+#ifndef _SURL_H_
+#define _SURL_H_
+
+#include <string>
+
+// A data Type to store a short URL and its expire time
+// url name wont be change after the time you created.
+// Example:
+//		SURL surl("https:://www.google.com", "2021/04/19");
+//		// Update the expire time after it is re-created
+//		surl.SetExpireTime(now);
+
+class SURL 
+{
+public:
+
+	// default constructor
+	SURL();
+
+	// default constructor with target url and now date
+	SURL(std::string url);
+
+	// default contructor with target url and specific expire date
+	SURL(std::string url, std::string expire);
+
+	// get the name of the SURL
+	std::string GetSURL();
+
+	// get the expire time of the SURL
+	std::string GetExpireTime();
+
+	// update the expire time of the SURL
+	std::string SetExpireTime(std::string date);
+private:
+
+	// the whole url of the SURL
+	std::string url;
+
+	// the expire date of the SURL
+	// Example: YYYY/MM/DD  -->  2021/04/19
+	std::string expire;
```

Comment:
Please consider better ways to represent the date, such as the boost datetime library:
https://www.boost.org/doc/libs/1_62_0/doc/html/date_time/gregorian.html#date_time.gregorian.date_class